server/lib/config/pbench-server-default.cfg

```diff
@@ -29,6 +29,20 @@ mailto=%(admin-email)s
 mailfrom=%(user)s@%(host)s
 commit_id=unknown
 
+# Configuration to support remote reverse-proxy configuration (e.g., NGINX)
+# in front of Pbench server.
+#
+# This will be advertised to front-end clients asking for endpoint configuration.
+# By default, this is the http port (80) on the server host, which will be
```

Comment:
Should this talk about being a host name / port pair?

Reply:
It's in the examples I gave; but you're right that I didn't say so directly up front, and that would be better... especially since I omitted the `:80` in the default value. (I didn't really think it would help to include that, but perhaps it would?)